src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -195,7 +195,7 @@ public Path getJavaAppEngineSdkPath() {
     return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
   }
 
-  private Path getJavaToolsJar() {
+  public Path getJavaToolsJar() {
```

Comment:
I wonder if these type of methods always return a `Collection<Path>` or `Path[]` for future-proofing (what happens if AppEngine's tools is split into multiple jars)?  

I think there are two approaches:
  1. The caller should know where the particular jars are situated within the Cloud SDK, in which case we just need a `resolve()` type method.  We're doing this in the Eclipse classpath providers, where we're pulling out runtime jars (servlet-api.jar and asp-api.jar).

  2. The locations are masked by this library, and the caller instead requests them by purpose, like this method.  Means we need to continue to modify and update this class as new libraries become included in the Cloud SDK.